internal/splunk/common.go

```diff
@@ -23,11 +23,14 @@ const (
 	DefaultSeverityTextLabel   = "otel.log.severity.text"
 	DefaultSeverityNumberLabel = "otel.log.severity.number"
 	HECTokenHeader             = "Splunk"
-	HecTokenLabel              = "com.splunk.hec.access_token" // #nosec
+	HTTPSplunkChannelHeader    = "X-Splunk-Request-Channel"
+
+	HecTokenLabel = "com.splunk.hec.access_token" // #nosec
```

Comment:
Tiny nit, the case of `Hec` should probably be consistent in these consts. One is `HEC`, the other two are `Hec`. (I realize it's not a part of this review, so feel free to ignore or update later.